src/main/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorUtils.java

```diff
@@ -1492,9 +1498,26 @@ public static String getBasesInWindowAroundReferenceAllele( final Allele refAlle
 
         final String referenceBases;
 
+        // Get the reference bases for this interval.
+        // This grossness is required for now as a fix for the hg19 vs b37 reference contig name issue:
```

Comment:
This hack can now be removed, as discussed in person.

Reply:
Fixed!